index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Technical Documentation Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>Product Landing Page</header>
+        <div>
+          <a href="#introduction" class="nav-link">Introduction</a>
+          <a href="#What_you_should_already_know" class="nav-link"
+            >What you should already know</a
+          >
+          <a href="#JavaScript_and_Java" class="nav-link"
+            >JavaScript and Java</a
+          >
+          <a href="#Variables" class="nav-link">Variables</a>
+          <a href="#Function_declarations" class="nav-link"
+            >Function declarations</a
+          >
+        </div>
+      </nav>
+
+      <section class="main-section" id="introduction">
+        <header>introduction</header>
```

Comment:
I see that you made great effort to re-phrase your review comment but it is still not clear what you mean. 
The question that will be on anyone's mind will be: 
- Which tag are you referring to?  "your tag should..." 

Your comment | Better re-phrase
--- | ---
your tag should wrap the toolbar or the nav not the title of sections. you could use the heading(h2,h3,h4,...) tags instead.  Typically the tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. | The (name the tag, e.g header tag `hx` ) should wrap.....

Kindly always make your comment with easy to understand sentences.